common/util/iframe.ts

```diff
@@ -0,0 +1,98 @@
+// Adopted from https://github.com/ueberdosis/tiptap/blob/main/demos/src/Experiments/Embeds/Vue/iframe.ts
```

Comment:
I'm not excited about having this file in `common/util/` instead of `web/components/editor/`, but it was the way I found to get parse.ts to be able to include this extension in `exhibitExts`

Reply:
maybe just rename file to "extension-iframe.ts" or "tiptap-extension-iframe.ts"?

Reply:
Hmm was wondering about more exotic options, like move exhibitExts into web/? Or somehow common/util/ just has the type definition, and the web implementation lives in web/? idk.

Renamed to tiptap-iframe for now